pages/video_tutorial.md

```diff
@@ -30,11 +30,23 @@ sitemap:
 </div>
 
 ## <i class="fa fa-film"></i> Other JHipster videos
-
 <div class="row">
     <div class="col-xs-12 col-sm-6 col-md-4">
         <div class="thumbnail no-margin-bottom">
             <div class="video-container">
+                <<iframe width="560" height="315" src="https://www.youtube.com/embed/F0LshPnnGY4" frameborder="0" allowfullscreen></iframe>
```

Comment:
Extra `<` before iframe tag. Also can you rebase against master, I added a video there as well so you might have to adjust the columns